src/Service/Sqs/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 - Enable compiler optimization for the `sprintf` function.
+- Add AddPermission endpoint
```

Comment:
should be moved in a new `### Added` section

(This is used to guess if next release should be a patch version (bug fix) or minor version (new features)